scriptworker/test/test_gpg.py

```diff
@@ -692,7 +599,15 @@ def test_verify_signed_git_commit_output_exception(output):
 
 
 @pytest.mark.asyncio
-async def test_get_git_revision_exception(mocker):
+async def test_get_latest_tag():
+    parent_dir = os.path.dirname(__file__)
+    expected = subprocess.check_output(["git", "describe", "--abbrev=0"], cwd=parent_dir)
```

Comment:
I'll merge this PR and file an issue for the test git repo.  It's easy to add; maybe I'll make this a good first bug.